### ISSUE TITLE
Tests fail on fresh clone (node v12)

### ISSUE THREAD

**Issue by kumavis** [ljharb/object.assign#76]
```
object.assign on  main is 📦 v4.1.0 via ⬢ v12.16.1 took 18s 
⇡0% ➜ yarn test
yarn run v1.22.4
$ npm run --silent lint && es-shim-api

/home/xyz/Development/object.assign/hasSymbols.js
  26:2   error    Invalid loop. Its body allows only one iteration  no-unreachable-loop
  33:22  warning  No magic number: 1                                no-magic-numbers

/home/xyz/Development/object.assign/implementation.js
  20:11  warning  No magic number: 1  no-magic-numbers

/home/xyz/Development/object.assign/polyfill.js
  39:18  warning  No magic number: 1  no-magic-numbers

/home/xyz/Development/object.assign/test/native.js
  22:56  warning  Unexpected 'this'  no-invalid-this

/home/xyz/Development/object.assign/test/shimmed.js
  25:56  warning  Unexpected 'this'  no-invalid-this

✖ 6 problems (1 error, 5 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by ljharb** [comment]
hmm, i can't reproduce this - that said, i don't use yarn, what does `npm test` do?

**Comment by ljharb** [comment]
nvm, with updated deps i can indeed repro it. On it.

**Comment by ljharb** [comment]
Fixed in f5cf6f1.
